app/src/pages/[resourceType]/[id].tsx

```diff
@@ -28,6 +31,45 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
     /* eslint-enable @typescript-eslint/no-explicit-any */
   }, []);
 
+  const [loadingIconVisible, setLoadingIconVisible] = useState(false);
+  const { data: dataRequirements, refetch } = trpc.service.getDataRequirements.useQuery(
+    { resourceType: jsonData.resourceType, id: jsonData.id as string },
+    { enabled: false }
+  );
```

Comment:
Small suggestion here - I've noticed that it is common practice to put `const`s at the beginning of the function, above the `useEffect`(s).  (See [app/src/pages/authoring/[resourceType]/[id].tsx](https://github.com/projecttacoma/measure-repository/blob/main/app/src/pages/authoring/%5BresourceType%5D/%5Bid%5D.tsx) as an example). I think we should put these two `const`s above the `useEffect` for the custom Prism rendering